hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/dao/GatewayServiceEntityRequestBuilder.java

```diff
@@ -67,19 +68,25 @@
                 .collect(flatten(MetricRequest::seriesRequests))),
         this.interactionRequestBuilder.build(entityRequest.incomingEdgeRequests()),
         this.interactionRequestBuilder.build(entityRequest.outgoingEdgeRequests()),
+        this.selectionConverter.convert(
+            entityRequest.labelRequest().isPresent()
+                ? Set.of(entityRequest.labelsAttributeRequest())
+                : Collections.emptySet()),
```

Comment:
nit: Can also use `.map().orElse()` on the Optional

Reply:
done